src/services/CicService.ts

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-console */
```

Comment:
Given all code is server-side, is a better alternative to update the es-lint rules for the project to disable this rule?